zqd/handlers_test.go

```diff
@@ -42,10 +42,39 @@ func TestSearch(t *testing.T) {
 	sp, err := client.SpacePost(context.Background(), api.SpacePostRequest{Name: "test"})
 	require.NoError(t, err)
 	_ = postSpaceLogs(t, client, sp.Name, nil, false, src)
-	res := zngSearch(t, client, sp.Name, "*")
+	_, res := zngSearch(t, client, sp.Name, "*")
 	require.Equal(t, test.Trim(src), res)
 }
 
+func TestSearchStats(t *testing.T) {
+	src := `
```

Comment:
Nit: const!

Reply:
We have `src := <string>`  throughout this file.